### PR TITLE
Add FLORES 200 test set

### PR DESCRIPTION
**Pull request by mkuchnik**
FLORES 200 dataset contained training set only. Now it has the test set.

### CODE REVIEW

File: datasets/flores-200/metadata.json [hunk 1]
```diff
@@ -215,6 +231,152 @@
           }
         }
       ]
+    },
+    {
+      "@type": "ml:RecordSet",
```
Comment (marcenacp) [inline datasets/flores-200/metadata.json:236]: Note: An alternate approach would have been to specify `"split": "dev" | "devtest"` as a parameter of all ml:RecordSets in order not to repeat them.

But because filtering is not implemented yet in mlcroissant, your current approach is probably the best.

**Review by marcenacp — APPROVED** [review]
Thanks1
